agents/supervisor/supervisor.go

```diff
@@ -119,11 +122,13 @@ func (s *Supervisor) AgentsList() []*agentlocalpb.AgentInfo {
 	res := make([]*agentlocalpb.AgentInfo, 0, len(s.agentProcesses)+len(s.builtinAgents))
 
 	for id, agent := range s.agentProcesses {
+		logs := agent.logs.GetLogs()
```

Comment:
in scope of this task we don't have requirement to split logs per agent, but ok.
QAN(bultin) agents logs are missed

Reply:
I'm surprised why it was made this way,  although there was suggested implementation with a separate endpoint for logs. This implementation doesn't include all pmm-agent related logs(like logs related to connection to pmm-managed).